### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by katiasuya**

### CODE REVIEW

File: internal/converter/converter.go [hunk 1]
```diff
@@ -33,29 +35,37 @@ func New(sem *semaphore.Weighted, repo *repository.Repository, storage *storage.
 }
 
 // Convert implements audio conversion.
-func (c *Converter) Convert(fileID, filename, sourceFormat, targetFormat, requestID string) {
+func (c *Converter) Convert(ctx context.Context, fileID, filename, sourceFormat, targetFormat, requestID string) {
+	ctxLogger, ok := mycontext.LoggerFromContext(ctx)
+	if !ok {
+		log.Fatal("can't get logger from context in Converter")
+		return
+	}
```
Comment (papahmsolo) [inline internal/converter/converter.go:43]: Fatal is for main

**Review by papahmsolo — COMMENTED** [review]
👀🙌

File: internal/logger/logger.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+type key int
+
+const loggerKey key = 0
+
+var DefaultLogger = &log.Logger{
+	Out:       os.Stdout,
+	Formatter: new(log.JSONFormatter),
+	Level:     log.InfoLevel,
+}
```
Comment (papahmsolo) [inline internal/logger/logger.go:19]: Is there a reason to make it exportable?

File: internal/server/handlers.go [hunk 3]
```diff
@@ -2,24 +2,28 @@
...
 	res "github.com/katiasuya/audio-conversion-service/internal/server/response"
 	"github.com/katiasuya/audio-conversion-service/internal/storage"
 	"github.com/katiasuya/audio-conversion-service/pkg/hash"
 )
 
-var errInvalidUsernameOrPassword = errors.New("invalid username or password")
+var (
+	errInvalidUsernameOrPassword = errors.New("invalid username or password")
+	errCantGetUserIDFomContext   = errors.New("can't get user id from context")
+)
```
Comment (papahmsolo) [inline internal/server/handlers.go:26]: What is the purpose of these error definitions? Are we handling these errors in any specific way?

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Logging part looks good to me

File: internal/server/handlers.go (hunk 3 above)
Comment (katiasuya) [reply internal/server/handlers.go:26]: I used to have various uses of this values, that's why I've created variables for them. But it's not necessary, you're right.

**Review by papahmsolo — APPROVED** [review]

**Pull request merged**